webui-src/app/main.js

```diff
@@ -26,30 +26,62 @@ const navIcon = {
 };
 
 const navbar = () => {
+  let toggleNav = false;
   return {
-    view: (vnode) => m(
-      'nav.tab-menu',
-      [
-        m('.nav-logo', [
-          m('img', {
-            src: '../data/retroshare.svg',
-            alt: 'retroshare_icon',
+    view: (vnode) =>
+      m('nav.menu', {}, [
+        m(
+          '.menu__logo',
+          {
+            style: { width: toggleNav ? '25px' : '130px' },
```

Comment:
If we create a `.collapsed` class, then all of the display styles can be added in scss itself.

Reply:
Ok, sure.

Reply:
Thanks for this suggestion, It really made the code more clean 😮.